app/components/organization/SettingsMenu.js

```diff
@@ -67,7 +67,7 @@ class SettingsMenu extends React.Component {
       {
         allowed: "organizationInvitationCreate",
         render: (idx) => (
-          <Menu.Button key={idx} href={`/organizations/${this.props.organization.slug}/users`} badge={this.props.organization.members.count + this.props.organization.invitations.count}>
+          <Menu.Button key={idx} link={`/organizations/${this.props.organization.slug}/users`} badge={this.props.organization.members.count + this.props.organization.invitations.count}>
```

Comment:
I was going to say that I think the needs to stay ah `href`, but `link` is fine. If you've got Teams enabled, and you click on the "Users" tab, you'll get taken to the new beta version - but that's totally fine anyway.